packages/marketplace/src/resources/locales/fr.json

```diff
@@ -0,0 +1,39 @@
+{
+  "translation": {
+    "settings": {
+      "optionsHeading": "Options",
+      "starCountLabel": "Nombre d\’étoiles",
```

Comment:
```suggestion
      "starCountLabel": "Nombre d’étoiles",
```
Unnecessary & invalid character escape
![image](https://user-images.githubusercontent.com/77577746/187638455-de7e9eb4-a939-4386-8006-0687fe97b4e9.png)

Reply:
I put the backslash `’` bcz some translations system need this symbols.
So if here it don't wan't the symbol to escape the character, remove it 👍🏻